package.json

```diff
@@ -27,19 +27,23 @@
     "path-is-absolute": "^1.0.0"
   },
   "devDependencies": {
-    "mocha": "~2.2.5",
+    "aliasify": "^2.1.0",
     "browserify": "~13.0.1",
     "browserify-prepend-licenses": "~1.0.0",
     "duck": "~0.1.11",
-    "uglify-js": "~2.4.8",
-    "temp": "~0.7.0",
     "eslint": "2.13.1",
+    "flow-bin": "^0.32.0",
     "hamjest": "2.13.0",
-    "flow-bin": "^0.32.0"
+    "mocha": "~2.2.5",
+    "temp": "~0.7.0",
+    "uglify-js": "~2.4.8"
   },
-  "browser": {
-    "./lib/unzip.js": "./browser/unzip.js",
-    "./lib/docx/files.js": "./browser/docx/files.js"
+  "browser": "mammoth.browser.js",
+  "aliasify": {
+    "aliases": {
+      "./unzip": "./browser/unzip.js",
+      "./files": "./browser/docx/files.js"
```

Comment:
Unfortunately, `aliasify` only compares `require()`'s argument directly without any notion of actual path, but for current use case it doesn't make any problems (at least, yet).